ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "ui-orion",
-  "version": "0.1.0",
+  "name": "@prefecthq/ui",
+  "version": "2.8.0",
```

Comment:
Nice. Are we going to try and keep this in sync with the prefect version?

Reply:
I'd love to if we could!